docs/_js/examples/todo.js

```diff
@@ -1,8 +1,8 @@
 var TODO_COMPONENT = `
 var TodoList = React.createClass({
   render: function() {
-    var createItem = function(itemText) {
-      return <li>{itemText}</li>;
+    var createItem = function(itemText, index) {
+      return <li key={index}>{itemText}</li>;
```

Comment:
I've always considered using just the index a React 'best practice', especially for lists that paginate so that you can re-use a lot of the DOM nodes in the list.  Isn't the reconciliation step wise enough to tell that only the new item exists on add, considering it is added to the end of the list, and most of the DOM manipulation will be skipped if just the index is used (I recognize this would be the same behavior as using the `itemText` in the key also)?